app/src/main/java/com/a1ishka/bankaccount/screens/newAccountScreen/InputWithTitle.kt

```diff
@@ -0,0 +1,54 @@
+package com.a1ishka.bankaccount.screens.newAccountScreen
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.OutlinedTextFieldDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+
+@Composable
+fun InputWithTitle(
+    @StringRes titleId: Int,
+    onTextChanged: (currentInput:String) -> Unit,
+    keyboardType: KeyboardType,
+    validated: Boolean
+) {
+    val borderColor = if (validated) {
+        Color.Gray
+    } else {
+        Color.Red
+    }
+
+    Text(
+        text = stringResource(titleId),
+        fontSize = 18.sp
+    )
+    OutlinedTextField(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 10.dp, bottom = 20.dp),
+        value = onTextChanged.toString(),
+        onValueChange = { onTextChanged },
```

Comment:
Does this lambda invoke? There might be "onValueChange = onTextChanged" or "onValueChange = { onTextChanged(it) }" to invoke lambda

Reply:
yes, understood. I changed it to "onValueChange = onTextChanged"